woocommerce/woocommerce-ios
# [Mobile Payments] Call out for support

Closes #4603 

This PR adds a call out for support specific to WCPay, that should only be available to those stores that are enrolled into WCPay and have In-Person Payments available.

Store enrolled into In-Person Payments:

| Before | After |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/126665309-e28d7355-2c95-49d3-b2dc-6be0d8a98234.jpeg" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/126665357-f7475c84-0a9a-48e2-a4b5-c973b26cf139.png" width="350"/> |


Store not enrolled into In-Person Payments (there are no user facing changes):

| Before | After |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/126665191-064f0f67-82f6-4ce5-bd1f-5e28fac9729d.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/126665191-064f0f67-82f6-4ce5-bd1f-5e28fac9729d.png" width="350"/> |

## Changes
* Touched HelpAndSupportViewController to add a new section that only gets displayed after checking if the store is eligible for In-Person Payments
* Touched ZendeskManager to tag the request properly. For context, see pbUcTB-kn-p2#comment-1333 and pdfdoF-L-p2

## How to test
* First, check that the call for support only shows up in stores eligible for In-Person Payments. To do so, navigate to Settings > Help & Support and check the contents of the table. Switch stores and test that the call out shows up in eligible stores, and does not show up in stores that are not setup to take In-Person Payments
* Second, check that the requests are tagged properly. Requests sent through the regular queue should be tagged exactly how the production app does. Requests sent through the WCPay queue should be [tagged like this one](https://woothemes.zendesk.com/agent/tickets/4166955) (see pbUcTB-kn-p2 for context)

Examples of tickets:
* Sent with develop (7.1) : [4167013-zd-woothemes]
* Sent from this branch via the default queue: [4166958-zd-woothemes]
* Sent from this branch via the WCPay queue: [4166955-zd-woothemes]

Update release notes:
